### DISCUSSION TITLE
Is it possible to connect to Kerberos enabled Elastic Search from Trino?

### DISCUSSION THREAD

**Discussion by mimi0524** [trinodb/trino#8453] [Q&A]
Hi All -

I am trying to use trino to  connect and query Kerberos enabled Elastic Search cluster. Is it possible to do so? Was unable to find  any properties related to kerberos in trino elastic connector. 

Appreciate your help on this !!

Thanks!!

**Comment by bitsondatadev** [comment]
Hey @mimi0524, unfortunately it's not supported at this time. We would need to investigate that to see what it would take. Feel free to open an issue. https://github.com/trinodb/trino/issues

**Reply by mimi0524** [reply]
Thanks for your response!!
